README.md

```diff
@@ -1,21 +1,57 @@
-# perl-Data-Resample
+# perl-Data-Decimate
 
-Data::Resample
+A module that allows you to decimate or reduce a data feed by selecting the last data in a given interval.
+See also https://en.wikipedia.org/wiki/Decimation_(signal_processing) .
 
-A module that allows you to resample a data feed
+#### SYNOPSIS
 
-sampling_frequency=15
-tick_cache_size = 30*60
-resample_cache_size = 12*60*60
-redis=xxxx
+```
+  use Data::Decimate qw(decimate);
 
-1) TICKS CACHE: Cache of all ticks in last tick_cache_size seconds
+  my @data = [
```

Comment:
This would create a list with 1 element which is an array of hashes. I think you either want to replace `[` by `(` or `@` by `$` and omit the `\` below.